### ISSUE TITLE
[BUG]: Primary key with UUID type is empty after record insert

### ISSUE THREAD

**Issue by metalrex100** [uptrace/bun#512]
After updating Bun from `v1.0.20` to `v1.1.3` insert queries not generating new uuid anymore for primary keys.

I have entity with field ID and bun tags: `bun:"type:uuid,default:uuid_generate_v4(),pk"` (same in SQL migration)

Previously empty field got new generated UUID value, but after update to `v1.1.3` it has just nil uuid value (all zeros).

**Comment by vmihailenco** [comment]
You also need to add `,nullzero` option. Perhaps, we should automatically apply `,nullzero` when we see `,default` option, but I am too scared to make any changes at this point :)

**Comment by metalrex100** [comment]
Ok, make sense, after changes to PK, I guess, it applies allow-zero strategy also to PK fields. Previously seems like PK always were considered as 'nullzero' values.

Add it to the doc at least for uuid type section please.
Probably for common PK topic also should be added.

**Comment by metalrex100** [comment]
And thank you for the fast response!)

**Comment by frederikhors** [comment]
> You also need to add `,nullzero` option. Perhaps, we should automatically apply `,nullzero` when we see `,default` option, but I am too scared to make any changes at this point :)

I think we should. Do not be afraid. There are changelogs for this.

**Comment by metalrex100** [comment]
@vmihailenco I added `,nullzero` tag to ID field, but it did not fix missing ID.
```
ID uuid.UUID `bun:"type:uuid,default:uuid_generate_v4(),pk,nullzero"`
```

**Comment by vmihailenco** [comment]
Seems to work for me - https://github.com/uptrace/bun/blob/master/internal/dbtest/pg_test.go#L522-L542

**Comment by metalrex100** [comment]
@vmihailenco found the issue. Once I rolled back to `v1.0.20`, `pgdialect` and `pgdriver` also rolled back to `v1.0.20`. So when I tried to reproduce `nullzero` solution with my PK with `v1.1.3` version it did not work because `pgdialect` and `pgdriver` packages were still `v1.0.20` and `pgdialect` return `feature.Returning` flag instead of `feature.InsertReturning`.
That is why in `Exec` method there were no scanning to model, just exec.

Updated everything to `v1.1.4`, everything works with `default` and without `nullzero`, thank you!
